### PR TITLE
add distribution metrics warning

### PR DESCRIPTION
**Pull request by maxday**
### What does this PR do?

Add details about custom metrics + warning about distribution
Add DD_TAGS config

### Motivation
Could lead to issues with customers

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by IvanTopolcic — APPROVED** [review]

File: content/en/serverless/azure_container_apps/_index.md [hunk 1]
```diff
@@ -82,18 +82,24 @@ See [Tracing Java Applications][1] for detailed instructions. [Sample code for a
 {{< /programming-lang >}}
 {{< /programming-lang-wrapper >}}
 
+### Custom metrics
+You can submit custom metrics using a [DogStatsd client][3].
+
+Please note that only `distribution` metrics should be used.
```
Comment (maycmlee) [inline content/en/serverless/azure_container_apps/_index.md:88]:
```suggestion
**Note**: Only `DISTRIBUTION` metrics should be used.
```

File: content/en/serverless/google_cloud_run/_index.md [hunk 2]
```diff
@@ -127,19 +127,24 @@ Below are instructions for deploying a Cloud Run service using standard GCP tool
 
    Under the **Environment variables** section, ensure that the name is set to `DD_API_KEY`.
 
+### Custom metrics
+You can submit custom metrics using a [DogStatsd client][5].
+
+Please note that only `distribution` metrics should be used.
```
Comment (maycmlee) [inline content/en/serverless/google_cloud_run/_index.md:133]:
```suggestion
**Note**: Only `DISTRIBUTION` metrics should be used.
```

**Review by maycmlee — COMMENTED** [review]
Hey @maxday! I made a couple of suggestions and fixed the links. Let me know if this works for you, thanks!

File: content/en/serverless/azure_container_apps/_index.md [hunk 3]
```diff
@@ -82,18 +82,24 @@ See [Tracing Java Applications][1] for detailed instructions. [Sample code for a
...
 | `DD_LOGS_ENABLED` | When true, send logs (stdout and stderr) to Datadog. Defaults to false. |
-| `DD_SERVICE` | See [Unified Service Tagging][4]. |
-| `DD_VERSION` | See [Unified Service Tagging][4]. |
-| `DD_ENV` | See [Unified Service Tagging][4]. |
-| `DD_SOURCE` | See [Unified Service Tagging][4]. |
+| `DD_SERVICE` | See [Unified Service Tagging][5]. |
+| `DD_VERSION` | See [Unified Service Tagging][5]. |
+| `DD_ENV` | See [Unified Service Tagging][5]. |
+| `DD_SOURCE` | See [Unified Service Tagging][5]. |
+| `DD_TAGS` | See [Unified Service Tagging][6]. |
```
Comment (maxday) [inline content/en/serverless/azure_container_apps/_index.md:102]: should this be [5] ?

Comment (maycmlee) [reply content/en/serverless/azure_container_apps/_index.md]: Yes, thank for catching that! Fixed it.

**Review by maycmlee — APPROVED** [review]
Thanks @maxday! Fixed the link.

**Pull request merged**